docs/modules/ROOT/pages/ios_accounts.adoc

```diff
@@ -0,0 +1,91 @@
+= Managing User Accounts
+:toc: right
+:toclevels: 1
+:sfauthenticationsession-url: https://developer.apple.com/documentation/safariservices/sfauthenticationsession
+:rfc-8252-url: https://tools.ietf.org/html/rfc8252#appendix-B.1
+
+The iOS app supports four ways of authenticating user accounts. 
+These are:
+
+* xref:basic-authentication[Basic Authentication]
+* xref:oauth-2-0[OAuth 2.0]
+* xref:safari-login-session[Safari Login Sessions]
+* xref:the-web-ui[The Web UI]
+
+== Authentication Options
+
+=== Basic Authentication
+
+* Login Credentials are stored on the device (secure in system keychain).
+* Password manager support.
+
+image:02_basic_auth.png[basic authentication]
+
+=== OAuth 2.0
+
+The OAuth2 implementation uses {sfauthenticationsession-url}[SFAuthenticationSession], which is described as a best practice by {rfc-8252-url}[RFC 8252]. 
```

Comment:
@settermjd `doc/SECURITY.md` got updated for iOs 12 in the `feature/sharing-user-groups` branch. (will be merged for initial release.

https://github.com/owncloud/ios-app/blob/feature/sharing-user-groups/doc/SECURITY.md#oauth2-implementation

Reply:
Was there a typo in that link, @michaelstingl? I can't find it anywhere.

Reply:
@settermjd got merged to master:
https://github.com/owncloud/ios-app/blob/master/doc/SECURITY.md#oauth2-implementation

Reply:
Improvement in 51de1bafc55eca5f9ebe7d3e7fcf5381af0bd376